File: submissions/sm_008_chaitanya/week_03/day_2/assignment/taxcalculator.js
var tax,rebate,income,savings;
var income = 700000;
var savings = 60000;
if(income < 250000){
    tax = 0;
    console.log(0)
}
else if(income >= 250000 && income < 500000){
    tax = (income-250000) * 0.1;
    console.log('tax:',tax)
}
else if(income >= 500000 && income <= 1000000){
    tax = (income- 250000) * 0.2;
    console.log('tax:',tax)
}
else if(income > 1000000){
    tax = (income-250000) * 0.3;
    console.log('tax:',tax)
}
if(income < 500000){
    rebate = savings*0.5;
    console.log('rebate:',rebate)
}
else if(income < 1000000){
    rebate = savings*0.3;
    console.log('rebate:',rebate)
}
else if(income > 1000000){
    rebate = savings*0.1;
    if(rebate > 50000){
        console.log('rebate:',50000)
    }
    else{
        console.log('rebate:',rebate)
    }
}

File: submissions/sm_108_krishna-kant/week_13/day_5/tax_calculator.py
print("Enter Total Income")
total_income = int(input())
tax = 0
taxable_income = 0
taxable_amount = 0
rebate = 0
print("Enter your savings")
savings = int(input())

#  Checking if savings is not greater than total income
if savings > total_income:
  print("Savings can't be more than total Income")

else :
#  Calculating Rebate
    if total_income < 500000 :
        rebate = 0.5 * savings

    elif total_income < 1000000 :
        rebate = 0.3 * savings

    elif total_income > 1000000:
        rebate = 0.1 * savings

#  Checking if rebate is more than capping
    if rebate > 50000: 
        rebate = 50000

# Calculating taxable income
taxable_income = total_income - rebate

#  Calculating Taxes
if taxable_income < 250000 :
    print("No tax applied!, Your total income is below taxation slab")

elif taxable_income > 250000 and taxable_income <= 500000: 
    taxable_amount = taxable_income - 250000
    tax = 0.1 * taxable_amount
    print( f" Total Income : {total_income} \n Rebate : {rebate} \n Taxable Amount : {taxable_amount} \n Tax Percentage : 10% \n Tax Amount : {tax}")

elif 500000 < taxable_income <= 1000000 :
    taxable_amount = taxable_income - 500000
    tax = 0.2 * taxable_amount + 0.1 * 250000
    print( f" Total Income : {total_income}  \n Rebate : {rebate} \n Taxable Amount : {taxable_amount}  \n Tax Percentage : 20% \n Tax Amount : {tax}")

elif taxable_income > 1000000 :
    taxable_amount = taxable_income - 1000000
    tax = 0.3 * taxable_amount + 0.2 * 500000 + 0.1 * 250000
    print( f" Total Income : {total_income}  \n Rebate : {rebate} \n Taxable Amount : {taxable_amount}  \n Tax Percentage : 30% \n Tax Amount : {tax}")